### DISCUSSION TITLE
Most Played problem

### DISCUSSION THREAD

**Discussion by davidegironi** [navidrome/navidrome#3073] [Q&A] [answered]
Hello,
I'm running navidrome (0.52.5) on Ubuntu 22.
I've a problem with Most Played.
There are a couple of album that seems to be "fixed" on top.
Is there a way to reset the "Most Played", or check why they are on top?

**Comment by caiocotts** [comment, marked as answer]
Hey there @davidegironi,

I was actually in a similar predicament myself recently.
You'll need to change the `play_count` column for those albums in the `annotation` table in the database.

**Reply by davidegironi** [reply]
Thank you. Dumb question, where can I found the database?

**Comment by caiocotts** [comment]
It would be inside the `data` directory. It's a file named `navidrome.db`.

**Reply by davidegironi** [reply]
Found in /var/lib/navidrome
Is it sqllite I suppose

**Reply by caiocotts** [reply]
That's correct.
You could do something like: `update annotation set play_count = 0 where item_id = <id of the album you want to reset>;`
This will set the play count of the album to 0.

DISCLAIMER
Please remember to back up your database before you make any manual changes, as you can make breaking changes.

**Reply by davidegironi** [reply]
What I've done
select * from annotation order by play_count DESC limit 10;
then my play_count was more than 100 for that couples of items. so
delete from annotation where play_count > 100;

Thanks
